packages/fxa-react/components/LogoLockup/index.tsx

```diff
@@ -3,28 +3,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { ReactElement } from 'react';
+import { useLocalization } from '@fluent/react';
 import logo from '../../images/ff-logo.svg';
 
 type LogoLockupProps = {
   children: string | ReactElement;
   className?: string;
 };
 
-export const LogoLockup = ({ children, className = '' }: LogoLockupProps) => (
-  <>
-    <img
-      src={logo}
-      data-testid="logo"
-      className="h-10 w-10 ltr:mr-4 rtl:ml-4"
-      alt="Firefox logo"
-    />
-    <h1
-      data-testid="logo-text"
-      className={`hidden tablet:inline-flex self-center text-xl ${className}`}
-    >
-      {children}
-    </h1>
-  </>
-);
+export const LogoLockup = ({ children, className = '' }: LogoLockupProps) => {
+  const { l10n } = useLocalization();
+  return (
+    <>
+      <img
+        src={logo}
+        data-testid="logo"
+        className="h-10 w-10 ltr:mr-4 rtl:ml-4"
+        alt={l10n.getString('app-logo-alt', null, 'Firefox logo')}
```

Comment:
This could be refactored to this format, used in other packages:

```
import { Localized } from '@fluent/react';

...
<Localized id='app-logo-alt'>
  <img
          src={logo}
          data-testid="logo"
          className="h-10 w-10 ltr:mr-4 rtl:ml-4"
          alt='Firefox logo'
  />
</Localized>

```

in `en.ftl`
```
# Alt text for logo 
app-logo-alt = 
  .alt = Firefox logo
```

Reply:
I have mixed feelings on it; I like the idea of having a rule of thumb that we should use `Localized` everywhere and only use `l10n.getString` when we need it or when it's just preferred for one reason or another, but I also like more simplified-looking FTL files.

Since we're using the `Localized` in other areas for the same string, maybe we should go with that? cc @bcolsson

Reply:
Thanks @flodolo, that's good to know.

@LZoog - I'll keep the existing strings as they are to minimize impact on l10n side, and moving forward prefer `<Localized>` for most new strings.